src/main/java/controller/signup/SignupState.java

```diff
@@ -19,6 +22,7 @@ public class SignupState {
     private String fullName = "";
     private String fullNameError;
 
+
     public String getUsername() {
```

Comment:
While adding new fields or methods (e.g. getters or setters), ensure that corresponding JavaDoc comments are added to improve the readability of the codebase :)